### ISSUE TITLE
Ender Dragon Addon

### ISSUE THREAD

**Issue by ElementalIdiot** [AlexModGuy/Ice-and-Fire-Suggestions#1180]
I understand plenty of people have already asked for this, but I think it would be cool if the ender dragon was updated to your style, and having it as an addon will give it less weight to the decision, meaning it can just be a little side thing with no particular incentive to rush it. The fact that it would be an addon will also allow for even more creative freedom if you are worried about it changing too much as well as giving people the OPTION to change the boss fight if they prefer the original one.

**Comment by M0ltenM3tal** [comment]
This actually wouldn't a bad idea. I didn't think about the other side if others didn't want a new ender dragon. Nice. Check issue #2 for ideas that people have come up with.

**Comment by CTH999** [comment]
Yeah, and maybe there could be weaker "end-dragons" that spawn naturally in the end. They are weaker,  always pitch black, fire out something like dragons breath but it doesn't linger. But, they can also cause some sort of blindness. They hoard a new metal, chorsite, which hase some uses. Not sure what.

They can also teleport short distances, and between dimensions if above size 3 (So you can ride them, and use them as a portable portal. If you enter into the end, which can only be done in the overworld, no matter where you exit, you exit in the place where you entered in the overworld. Same with entering the end. Enter it, and you will apear at the same point where you left. However, in the nether, the same properties apply as normal, allowing for rapid travel!

Note: When teleporting, they throw something from their mouth that looks like an enterpearl, but won't damage them. It will damage whoever it hits though!

Ender steel is immune to void damage, and will hover over the void. It is also immune to lava. But, it is made in that furnace with netherite ingots, not iron. Ender-steel swords deal some void damage along with regular damage (1 guarenteed hert plus a bit more damage than a lightningsteel sword) The full set makes it so you can't fall bellow y=0 (Or y=-64 in 1.18), so you can't hit the void! (If you are wearing the full armour set)

**Comment by LAB5565** [comment]
You know, I did see a ender dragon add on for Ice and Fire on curseforge. Not sure how good it is, but it might be worth a shot
